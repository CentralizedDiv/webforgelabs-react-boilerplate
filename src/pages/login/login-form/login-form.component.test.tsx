import React from 'react';
import LoginForm from './login-form.component';
import { shallow } from 'enzyme';

describe('LoginForm', () => {
  const component = shallow(<LoginForm/>);

  it('renders properly', () => {
    expect(component).toMatchSnapshot();
  });

});