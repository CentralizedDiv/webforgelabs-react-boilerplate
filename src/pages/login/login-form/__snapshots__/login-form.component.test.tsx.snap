// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm renders properly 1`] = `
<div
  className="st-login-page__login-form-container"
>
  <div
    className="login-form-container__form"
  >
    <div
      className="form__title"
    >
      <span>
        pages.login.title
      </span>
    </div>
    <div
      className="form__subtitle"
    >
      <span>
        pages.login.subtitle
      </span>
    </div>
    <Form(FormInputs) />
  </div>
</div>
`;
